pkg/validate/validate.go

```diff
@@ -18,6 +18,9 @@ import (
 	"k8s.io/apimachinery/pkg/util/validation"
 )
 
+// IBMCloudClusterNameMaxLength is the maximum allowed length for a cluster name on IBM Cloud
+const IBMCloudClusterNameMaxLength = 32
```

Comment:
Please add to this comment for how this number is determined.

Reply:
Oh, this was based on an IBM Kubernetes restriction, which isn't applicable here.
I shall add the formula and new value, once I work that out calculation.